src/bp/ui-studio/src/web/views/OneFlow/SidePanel/typings.d.ts

```diff
@@ -0,0 +1,26 @@
+import sdk from 'botpress/sdk'
+import { FlowView } from 'common/typings'
+
+export interface ImportActions {
```

Comment:
It seems like this type should be singular instead of plural:

```suggestion
export interface ImportAction {
```